bower.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "json-refs",
   "main": "browser/json-refs.js",
-  "version": "1.3.0",
```

Comment:
bower requires tags (so it is redundant) while npm does not. Here's where it is mentioned that version is now deprecated for bower.json: https://github.com/bower/spec/blob/master/json.md#user-content-version . As far as npm, you only need to update package.json's `version` before publishing, but with bower, you should just add a tag and push tags to github (but if you want to keep version for some reason, then you'll also need to update it before making a tag).